Backpack/StarRating/Classes/BPKStar.m

```diff
@@ -0,0 +1,160 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKStar.h"
+#import <Backpack/Color.h>
+#import <Backpack/Common.h>
+#import <Backpack/Icon.h>
+#import <Backpack/BPKRTLSupport.h>
+
+@interface BPKStar ()
```

Comment:
This file should also use `NS_ASSUME_NONNULL_BEGIN`/`NS_ASSUME_NONNULL_END`

Reply:
Done